src/screens/TopicMemberScreen/index.js

```diff
@@ -165,10 +167,9 @@ const TopicMemberScreen = () => {
         getSearchLayout={setSearchHeight}
       />
       <TopicMemberHeadline text="Visible members of this community" />
-      <UsersFragmentCommunity
+      <UsersFragment
         isLoadingDiscoveryUser={isLoadingDiscovery.user}
-        isFirstTimeOpen={isFirstTimeOpen}
-        initialUsers={initalMember}
+        isFirstTimeOpen={false}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `UsersFragment` component is now being used with a hardcoded `isFirstTimeOpen` prop value of `false`, replacing the previous dynamic value. If this prop significantly affects the component's behavior, this change could have a substantial impact. Also, the `initialUsers` prop has been removed. Ensure these changes are intentional and tested.
```diff
- <UsersFragmentCommunity
-    isFirstTimeOpen={isFirstTimeOpen}
-    initialUsers={initalMember}
+ <UsersFragment
+    isFirstTimeOpen={false}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->